apicapi/tests/unit/test_apic_manager.py

```diff
@@ -262,7 +262,7 @@ def test_ensure_infra_created_seq1(self):
 
         switch_dict_copy = copy.deepcopy(self.mgr.switch_dict)
         for value in switch_dict_copy.values():
-            for key in value.keys():
+            for key in list(value.keys()):
```

Comment:
Is this list() still needed?

Reply:
Yes, because we are deleting from values, it raises "RuntimeError: dictionary changed size during iteration" without wrapping of `list`. By wrapping up with list, we create a copy of `dict_keys` as `list`. So it avoids the above Runtime error.